src/main/resources/application.properties

```diff
@@ -5,4 +5,5 @@ spring.datasource.password=kodilla_password
 spring.jpa.database=mysql
 spring.jpa.show-sql=true
 spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.naming.strategy=org.hibernate.cfg.ImprovedNamingStrategy

```

Comment:
Ja bym to wyrzucił, Marcin wrzucił wczoraj zaktualizowanego commita application.properties

Reply:
to jest niepotrzebne